.eslintrc.json

```diff
@@ -3,18 +3,20 @@
     "parser": "@typescript-eslint/parser",
     "plugins": [
         "@typescript-eslint",
-        "react-hooks"
+        "react-hooks",
+        "iot-cardboard-js"
     ],
     "extends": [
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
         "prettier",
-        "prettier/@typescript-eslint"
+        "plugin:iot-cardboard-js/recommended"
```

Comment:
ohh do we have a plugin now? Can you walk us through about those eslint related changes?

Reply:
I wrote it on Friday. I can certainly go over it for folks that are interested.